bears/general/InvalidLinkBear.py

```diff
@@ -24,8 +25,18 @@ def get_status_code_or_error(url):
         except requests.exceptions.RequestException:
             pass
 
-    @staticmethod
-    def find_links_in_file(file):
+    def get_status_code_checked(self, url):
+        code = InvalidLinkBear.get_status_code(url)
+        if code is None:  # pragma: no cover
+            # if status code of google.com is None, then its
+            # an unstable connection.
+            if self.get_status_code(self.check_connection_url) is None:
+                self.err("Unstable connection detected. "
+                         "InvalidLinkBear will not continue.")
```

Comment:
Please use `self.__class__.__name__` so that we dont have to change this if the bear's name changed